### PR TITLE
Update staging and production URLS after the move from Heroku

### PR DESCRIPTION
**Pull request by tahb**
## Changes in this PR

Update to the Readme before we turn off Heroku.

## Next steps

- [ ] Is an ADR required? An ADR should be added if this PR introduces a change to the architecture.
- [ ] Is a changelog entry required? An entry should always be made in `CHANGELOG.md`, unless this PR is a small tweak which has no impact outside the development team.
- [ ] Do any environment variables need amending or adding?
- [ ] Have any changes to the XML been checked with the IATI validator? See [XML Validation](https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/develop/doc/xml-validation.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,13 +98,15 @@ The heading should link to a Github URL at the bottom of the file, which shows t
...
+The app is currently hosted on GPaaS: [https://beis-roda-staging.herokuapp.com/](https://beis-roda-staging.london.cloudapps.digital)
 
 The `develop` branch is deployed to staging after a successful build via Travis CI.
 
 ### Production
 
-TBC
+The app is currently hosted on GPaaS: [https://beis-roda-staging.herokuapp.com/](https://beis-roda-prod.london.cloudapps.digital)
+
+The `master` branch is deployed to staging after a successful build via Travis CI.
```
Comment (lozette) [inline README.md]: `staging` should be `prod`!

**Comment by mec** [comment]
Don't forget the link at the top of the Github repo as well!

File: README.md (hunk 1 above)
Comment (tahb) [reply README.md]: 🤦‍♂  thanks!

Comment (tahb) [reply README.md]: @lozette updated 👍

File: README.md [hunk 2]
```diff
@@ -86,13 +86,15 @@ To manage sensitive environment variables:
...
+The app is currently hosted on GPaaS: [https://beis-roda-staging.london.cloudapps.digital](https://beis-roda-staging.london.cloudapps.digital)
 
 The `develop` branch is deployed to staging after a successful build via Travis CI.
 
 ### Production
 
-TBC
+The app is currently hosted on GPaaS: [https://beis-roda-prod.london.cloudapps.digital](https://beis-roda-prod.london.cloudapps.digital)
+
+The `master` branch is deployed to staging after a successful build via Travis CI.
```
Comment (lozette) [inline README.md]: This still says staging! It should read 
```The `master` branch is deployed to production after a successful build via Travis CI.```

Comment (tahb) [reply README.md]: I corrected the URLs and missed this bit as well. Updated again!

**Pull request merged**